### PR TITLE
FF118Relnotes: RTCRtpTransceiver.direction/currentDirection supported

### PR DESCRIPTION
**Pull request by hamishwillee**
[`RTCRtpTransceiver.currentDirection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver/currentDirection) and [`RTCRtpTransceiver.direction`](https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver/direction)  are supported in FF118. This just adds a release note. 

Related docs work can be tracked in #28845

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/118/index.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ This article provides information about the changes in Firefox 118 that affect d
 
 ### APIs
 
+- The {{domxref("RTCRtpTransceiver.currentDirection")}} and {{domxref("RTCRtpTransceiver.direction")}} properties now support the `"stopped"` value for indicating whether a transceiver has stopped, and should be used instead of the deprecated {{domxref("RTCRtpTransceiver.stopped")}} property ([Firefox bug 1568296](https://bugzil.la/1568296)).
+- The array returned by {{domxref("RTCPeerConnection.getTransceivers()")}} now omits stopped transcievers. Similarly, {{domxref("RTCPeerConnection.getReceivers()")}} and {{domxref("RTCPeerConnection.getSenders()")}} omit the receivers and senders, respectively, associated with a stopped transceiver ([Firefox bug 1568296](https://bugzil.la/1568296)).
```
Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/118/index.md:44]: FYI, this is same bug and is "related functionality". But it is distinct APIs affected, so have created a separate line item.

Comment (hamishwillee) [inline files/en-us/mozilla/firefox/releases/118/index.md:44]:
```suggestion
- The array returned by {{domxref("RTCPeerConnection.getTransceivers()")}} now omits stopped transcievers. Similarly, {{domxref("RTCPeerConnection.getReceivers()")}} and {{domxref("RTCPeerConnection.getSenders()")}} omit the receivers and senders associated with a stopped transceiver ([Firefox bug 1568296](https://bugzil.la/1568296)).
```

**Review by dipikabh — APPROVED** [review]
Looks good, thank you 👍

**Comment by dipikabh** [comment]
Oh, this'll need merge conflict resolution

**Comment by hamishwillee** [comment]
THanks for the review @dipikabh !

**Pull request merged**